tests/e2e/ansible/install_kubeadm.yml

```diff
@@ -46,6 +46,8 @@
         url: https://storage.googleapis.com/kubernetes-release/release/{{ k8s_version }}/bin/linux/{{ target_arch  }}/{{ item }}
         dest: /usr/local/bin
         mode: '+x'
+      retries: 3
```

Comment:
hi @ldoktor ! Good to know what ansible has now retry for tasks (or exists a long time and I never noticed:(

I noticed that on this file you didn't retry `unarchive` tasks. Isn't it idempotent?